packages/evolution-common/src/services/surveyGeneration/SurveyGenerator.ts

```diff
@@ -0,0 +1,21 @@
+import { SurveyWidgetConfig } from "../../../../../example/demo_survey/src/config/widgets.config";
+import { readCSVFile } from './CSVHandle';
+
+
+export async function WidgetListGenerator (filePath: string): Promise<SurveyWidgetConfig[]> {
+    const widgetList : SurveyWidgetConfig[] = [];
+    await readCSVFile(filePath)
+        .then((data) => {
+            data.length !== 0 && data.forEach((widgetProperties) => {
+                const widget : SurveyWidgetConfig = { ...widgetProperties as SurveyWidgetConfig };
```

Comment:
There is no preparation or validation of the data, it's just the csv as is used for widget properties?